client/signup/navigation-link/test/index.jsx

```diff
@@ -77,6 +80,11 @@ describe( 'NavigationLink', () => {
 	test( 'should set a proper url as href prop when the direction is "back".', () => {
 		expect( getStepUrl ).not.toHaveBeenCalled();
 
+		getFilteredSteps.mockReturnValue( {
```

Comment:
getFilteredSteps returns an array, not an object - thus this tests for the wrong type of value

Reply:
Also it would be great to have some tests for getFilteredSteps itself - this PR seems to revolve around it, but it has no coverage

Reply:
Good point, I'll add some today, thanks @adamziel !

Reply:
Thanks for spotting that! - `getFilteredSteps` should now be pretty well covered in https://github.com/Automattic/wp-calypso/pull/35049/commits/def07a343be80fb790ad6c773eb35c5bd991346b